usaspending_api/common/experimental_api_flags.py

```diff
@@ -0,0 +1,20 @@
+"""
+Some API endpoints accept "X-Experimental-API" as a HTTP Header to control experimental API functionality.
+The constants expected are defined in this file to insure consistency across different endpoints that may / will
+need to have experimental functionality alongside the expected functionality as the changes are made permanent
+in Production.
+
+EXPERIMENTAL_API_HEADER - HTTP Header expected to access experimental functionality.
+ELASTICSEARCH_HEADER_VALUE - Value expected to access Elasticsearch functionality on specific endpoints.
+"""
+from rest_framework.request import Request
+
+EXPERIMENTAL_API_HEADER = "HTTP_X_EXPERIMENTAL_API"
+ELASTICSEARCH_HEADER_VALUE = "elasticsearch"
+
+
+def is_experimental_elasticsearch_api(request: Request) -> bool:
+    """
+    Returns True or False depending on if the expected_header_value matches what is sent with the request.
+    """
+    return request.META.get(EXPERIMENTAL_API_HEADER) == ELASTICSEARCH_HEADER_VALUE
```

Comment:
If we have multiple experimental things in play, should the value for this header instead be a `list`? And the check be whether the list includes the particular experimental feature that the user is trying to use?

Reply:
I like the idea, but it would require passing the headers as some sort of formatted string such as "header1/header2/header3" because I don't believe HTTP Headers will take a list as a valid value. It seems to convert the values to strings.